src/picologging/_picologging.cxx

```diff
@@ -12,8 +13,39 @@
 #include "handler.hxx"
 #include "streamhandler.hxx"
 
+std::map<unsigned short,std::string> LEVELS_TO_NAMES = {
```

Comment:
One question/point is that if the [addLevelName](https://docs.python.org/3/library/logging.html#logging.addLevelName) is added, will it be ok to assign to this map as I guess it's not thread-safe. If that's ever going to be done...

Reply:
use `unordered_map` instead of `map` since the order is irrelevant.

You can also mark this as `const`

Reply:
Done.